Frontend/Tumultu/src/app/file/list/components/files-list/files-list.component.html

```diff
@@ -0,0 +1,7 @@
+<mat-list style="width: 90%">
+  <div *ngFor="let file of files | async">
```

Comment:
just an Idea, we can implement new angular syntax for iterations in .html file:

@for (file of files | async; ) {
  <mat-list-item>....
}

New syntax provides easier methods to implement empty list alerts and many different useful templates